org/mergeOnGreen.ts

```diff
@@ -30,12 +31,14 @@ export const rfc10 = async (status: Status) => {
     const issue = await api.issues.get({ owner, repo, number })
 
     // Get the PR combined status
-    const mergeLabel = issue.data.labels.find((l: LabelLabel) => l.name === "Merge On Green")
+    const issueLabelNames = issue.data.labels.map(l => l.name)
+    const mergeLabel = Object.values(labelMap).find(label => issueLabelNames.includes(label.name))
+
     if (!mergeLabel) {
-      return console.log("PR does not have Merge on Green")
+      return console.log("PR does not have Merge on Green-type label")
     }
 
-    let commitTitle = `Merge pull request #${number} by Peril`
+    let commitTitle = `${capitalize(mergeLabel.mergeMethod)} pull request #${number} by Peril`
```

Comment:
If squashing, I actually think it's better to leave the default commit title, which is the title of the PR, with the PR number added on the end. When you squash, all the other commits will disappear from the history, so it's important that the single, squashed commit has an informative title (the titles of the squashed commits are then by default included as a bullet list in the commit description).

Reply:
@admbtlr do you feel that this is blocking? I'm inclined to merge this PR and iterate but I'm not an experienced squasher, so wasn't sure. Either way, I have ideas about how I might take these abstractions further and let each strategy define how they merge.

Reply:
Ahhhh right, that makes sense! Thank you, I will follow up.

Reply:
I've added a new commit to handle this: https://github.com/artsy/peril-settings/pull/161/commits/ae4349b0d18ed31716829d7dc24ed3b7efe45d82

I had to amend an earlier commit that updated our dependencies, since Prettier was unaware of some syntax I used.